src/types/featureFlags.ts

```diff
@@ -8,6 +8,7 @@ export interface FeatureFlags {
     message: string
   }
   [FEATURE_FLAGS.NPS_FORM]: boolean
+  [FEATURE_FLAGS.ENABLED_BLOCKS]: string[]
```

Comment:
if this is only for homepage, can we name as HOMEPAGE_ENABLED_BLOCKS?

Reply:
+1

Reply:
done